### PR TITLE
bug: fixed splash screen styling on small screens

### LINKED ISSUE #1384: [BUG] Improve splash screen sizing for narrow devices
Splash screens look great on sensible screen dimensions. When changing the screen to be very narrow (we're considering 320px as the minimal screen width) the components do not behave the same as they do in a regular template: 
- text does not wrap, instead it runs off the screen on the right hand side
- image does not shrink, instead it runs off the screen on the right hand side

See terms and conditions page that launches after the intro for comparison. 

https://user-images.githubusercontent.com/74557272/169017896-064a32cb-f12e-4461-aac6-f7f5f15129f0.mp4

_Originally posted by @EsmeeIDEMS in https://github.com/IDEMSInternational/parenting-app-ui/pull/1351#pullrequestreview-976720590_

**Screenshot on screen of 320 px width**
image

### PR DESCRIPTION
**Pull request by jfmcquade**
PR Checklist

- [x] - Latest `master` branch merged
- [x] - PR title descriptive (can be used in release notes)

## Description

Fixes but where splash screen slideshow didn't display properly on small screens, with some content being cut off (see issue #1384).
Also makes `display-group` with the style `two_columns_images` change its padding responsively, making the grid spacing smaller for screens less than 375px in width. Also renames 

## Git Issues

Closes #1384 

## Screenshots/Videos

The splash screens resizing responsively, displaying adequately on screens as small as 320px (the smallest size we support)

https://user-images.githubusercontent.com/64838927/174628349-d1d36cb3-7512-427a-abf8-615fa7d7fdd7.mov

### CODE REVIEW

File: packages/app-data/sheets/template/splash_screens.json [hunk 1]
```diff
@@ -294,7 +297,7 @@
             {
               "type": "display_group",
               "parameter_list": {
-                "style": "two_columns_images tight"
+                "style": "two_columns_images grid-spacing-sm"
```
Comment (chrismclarke) [inline packages/app-data/sheets/template/splash_screens.json:300]: naming convention feels very sensible to me

File: src/theme/deployment/components/_display-group.scss [hunk 2]
```diff
@@ -50,11 +50,18 @@ plh-tmpl-display-group .display-group-wrapper[data-param-style~="parent_point"]
...
+  @media (max-width: 375px) {
     plh-template-component {
-      display: block;
-      max-height: 70px;
-      max-width: 160px;
+      padding: 4px;
+    }
+  }
+  &[data-param-style~="grid-spacing-sm"] {
+    grid-gap: 16px;
```
Comment (chrismclarke) [inline src/theme/deployment/components/_display-group.scss:59]: looks good to me, little chance of unintended knock-ons when using selectors in this way.

**Review by chrismclarke — APPROVED** [review]
Visually looks good to me and code makes sense, I've added one comment for a proposed follow-up issue, but otherwise I'd say good to go!

**Review by EsmeeIDEMS — APPROVED** [review]
Functional test passed.

**Pull request merged**